test/functional/ios/crash-specs.js

```diff
@@ -17,7 +17,7 @@ describe('crash recovery @skip-real-device', function () {
       .elementByAccessibilityId("Crash")
       .click()
       .then(function () {
```

Comment:
@moizjv this can just be `.sleep(3000)`, and we always include a comment about why we're using a static wait